pkg/config/errors.go

```diff
@@ -33,6 +33,23 @@ func (e BpError) Unwrap() error {
 	return e.Err
 }
 
+// HintError wraps another error to suggest other values
+type HintError struct {
+	Hint string
+	Err  error
+}
+
+func (e HintError) Error() string {
+	if len(e.Hint) > 0 {
```

Comment:
Let's not do string substitution here. 
Instead of `HintError{"suggested_moduleid", errr}`, do `HintError{'Did you mean "suggested_moduleid"', errr}`

Reply:
Done